src/lib.rs

```diff
@@ -955,12 +962,14 @@ where
                 client_secret.clone(),
                 self.issuer.clone(),
                 self.jwks.clone(),
+                self.id_token_signing_algs.clone(),
```

Comment:
We can avoid breaking changes here by just calling `set_allowed_algs` on the new verifier before returning it